README.md

```diff
@@ -308,3 +308,9 @@ For community packages and providers, we currently are promoting a community pro
 **Note that all of these plugins/providers/packages are currently for v3**
 
 - [NgxStrapiAuth](https://github.com/jabali2004/ngx-strapi-auth) - Angular authentication library with pre-built components.
+
+## Notice
+
+As of January 2022, this repo is no longer maintained by Strapi LLC. nor Strapi SAS and the ownership has been collectively transferred to the Strapi Community who has no directly relation to either Strapi LLC. nor Strapi SAS. Strapi is a registered trademark of Strapi LLC and the Strapi Community have been granted exclusive inherited rights to use the name "Strapi" as well as the Strapi logo contained in the following files: [awesome-strapi-logo.jpg](./media/awesome-strapi-logo.jpg) and [awesome-strapi-logo.png](./media/awesome-strapi-logo.png). Further modification of this logo is allowed under the MIT license stored in this repo by the new maintainers.
+
+If anyone has any questions regarding the transfer of the repo or the rights granted please feel free to email Strapi LLC directly at hi@strapi.io
```

Comment:
Who is supposed to answer the inquiries? Should we create a specific email address or use the forum for these?

Reply:
We have a community@strapi.io we could use instead. It comes to the support team but I think any questions on this would likely be legal in nature. I'd find it highly unlikely we would get any misc questions.

Reply:
I see either way works